.travis.yml

```diff
@@ -46,9 +50,9 @@ after_success:
 
 deploy:
```

Comment:
You might want to have a separate stage for deploy, which will only run after all tests.

```yaml
jobs:
  include:
  - stage: Deploy
    before_install: []
    install: skip
    script: skip
    after_success: []
    deploy:  # move it here
```